sentry-core/src/main/java/io/sentry/core/UncaughtExceptionHandlerIntegration.java

```diff
@@ -20,24 +21,24 @@
 public final class UncaughtExceptionHandlerIntegration
     implements Integration, Thread.UncaughtExceptionHandler, Closeable {
   /** Reference to the pre-existing uncaught exception handler. */
-  private Thread.UncaughtExceptionHandler defaultExceptionHandler;
+  private @Nullable Thread.UncaughtExceptionHandler defaultExceptionHandler;
 
-  private IHub hub;
-  private SentryOptions options;
+  private @NotNull IHub hub;
+  private @NotNull SentryOptions options;
```

Comment:
m: I think this can't be `@NotNull` since the value will be always null once we construct the object.

Reply:
good catch